packages/components/src/DateTimePickers/InputDateTimePicker/date-extraction.js

```diff
@@ -48,23 +51,28 @@ function checkTime({ hours, minutes }) {
 	if (minutes.length !== 2 || isNaN(minsNum) || minsNum < 0 || minsNum > 59) {
 		throw new Error('TIME - INCORRECT MINUTES NUMBER');
 	}
+	const secondsNum = Number(seconds);
+	if (seconds.length !== 2 || isNaN(secondsNum) || secondsNum < 0 || secondsNum > 59) {
+		throw new Error('TIME - INCORRECT SECONDS NUMBER');
+	}
 }
 
 /**
- * Convert hour and minutes into minutes
+ * Convert hour minutes and seconds into seconds
  * @param hours {string}
  * @param minutes {string}
+ * @param seconds {string}
  * @returns {number}
  */
-function hoursAndMinutesToTime(hours, minutes) {
-	checkTime({ hours, minutes });
-	return Number(hours) * 60 + Number(minutes);
+function hoursAndMinutesAndSecondsToTime(hours, minutes, seconds) {
```

Comment:
Perhaps you can change the naming :D `timeToSeconds`

Reply:
right haha